openfecwebapp/templates/macros/filters/date.html

```diff
@@ -46,7 +46,6 @@
   </div>
   <div class="filter js-filter" id="{{ name }}" data-filter="date" data-name="{{ name }}" data-validate="true">
     <fieldset>
-    <legend class="label">{{ label }}</legend>
```

Comment:
Removing the `<legend>` we also need to remove the `<fieldset>` which now that I think about it, shouldn't be necessary.